geonode_mapstore_client/client/js/plugins/MapViewersCatalog.jsx

```diff
@@ -112,7 +113,7 @@ function MapViewersCatalogPlugin({
                     <div className="gn-new-map-viewer-action-wrapper">
                         <div  className="gn-new-map-viewer-action-card">
                             <div>
-                                <Icon glyph="link" className="fa-4x"/>
+                                <Glyphicon glyph="link" className="fa-4x"/>
```

Comment:
Do we need the fa-4x class? Could we try to use the Text component with font size xl as wrapper instead?

Reply:
Not needed, likely overlooked during the bulk replacement. Thanks for catching it!

Reply:
Updated